src/CADL.Extension/Emitter.Csharp/src/emitter.ts

```diff
@@ -161,12 +170,16 @@ export async function $onEmit(
                 resolvePath(options.csharpGeneratorPath, "..", "Azure.Core.Shared"),
             ]
             for (const sharedFolder of sharedFolders) {
-                resolvedSharedFolders.push(path.relative(generatedFolder, sharedFolder));
+                resolvedSharedFolders.push(path.relative(generatedFolder, sharedFolder).replaceAll("\\", "/"));
             }
      
             if (!fs.existsSync(generatedFolder)) {
                 fs.mkdirSync(generatedFolder, { recursive: true });
             }
+            
+            if (options["clear-output-folder"]) {
```

Comment:
Have you tried inputting an arbitrary value to `options["clear-output-folder"]`? I think we should make it robust to `options["clear-output-folder"] === true`.

Reply:
I think it make if false be default, and will let customer to decide.

Reply:
I mean if customer pass in "aaa"

Reply:
I think we have set the type 'boolean' in schema, it should not pass 'aaa'

Reply:
Yes, boolean is expected, but user could do anything. What if they pass in 'aaa'?

Reply:
I mean the schema will make sure to set the boolean value, otherwise the emitter will crush because of invalid-schema (error invalid-schema: Schema violation: must be boolean (/clear-output-folder)).